3 - PB/Documentazione interna/Verbali/24-03-30/24-03-30.typ

```diff
@@ -0,0 +1,104 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "30/03/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Oseliero Antonio",
+  ),
+  reviewers: (
+    "Banzato Alessio",
+  ),
+  missingMembers: (
+    "Todesco Mattia",
+  ),
+  timeStart: "10:30",
+  timeEnd: "11:15",
+);
+
+= Ordine del giorno
+- Valutazione del progresso generale;
+- Pianificazione.
+- Invio candidatura PB;
+
+= Valutazione del progresso generale <avanzamento>
+
+Lo Sprint 21 è servito al gruppo per pianificare e continuare gli ultimi lavori in previsione del colloquio PB.
+
+== #ndp
+
+Sono state aggiornate e revisionate le seguenti sezioni:
+- Processo di gestione della qualità;
+- Processo di validazione;
+- Processo di integrazione;
+- Processo di verifica;
+- Processo di implementazione;
+- Processo di transizione.
+
+== #pdq
+
+- redatto tracciamento test;
+- aggiornate metriche dalla dashboard.
+
+== #st
+
+- completata e ampliata la specifica tecnica.
+
+== #man
+
+- definite le ultime task;
+- redatta sezione Modifica Zona;
+- redatta sezione Ordine Movimentazione Ordine;
```

Comment:
C'è un typo ma in ogni caso la sezione si chiama "Richiesta spostamento prodotto"